src/main/electron/listener.cljs

```diff
@@ -125,6 +127,26 @@
                      (fn [code]
                        (user/login-callback code)))
 
+  (js/window.apis.on "quickCapture"
+                     (fn [args]
+                       (let [{:keys [url title description]} (bean/->clj args)
+                             time (date/get-current-time)
+                             text (if (not-empty description) (str (string/trim description) " via" ) "")
+                             template "**{time}** [[quick capture]]: {text} [{title}]({url})"
```

Comment:
Could we make this configurable via a param e.g. `(or template "**{time}** [[quick capture]]: {text} [{title}]({url})")`? I like this functionality but would prefer to import data as properties e.g. `"title:: {title}\nurl:: {url}"` . I can push up a commit for this if you would like

Reply:
Thinking on this more, how about a simpler API of a `content` param that can handle `{time}`? This gives users maximum functionality and less for us to maintain

Reply:
Will use the same style as mobile share intent.

https://github.com/logseq/logseq/blob/master/src/main/frontend/mobile/intent.cljs#L93-L96

Reply:
👍